bin/notify-slack.sh

```
@@ -0,0 +1,128 @@
+⚠️ Base images are NOT ready - manual intervention required"
+fi
+
+echo "Sending Slack notification for $ENVIRONMENT environment..."
+echo "Status: $JOB_STATUS"
+echo "Channel: $SLACK_CHANNEL"
+echo "Message preview:"
+echo "$SLACK_MESSAGE"
+
+# Send to Slack using webhook with enhanced formatting
+if [ -n "$SLACK_WEBHOOK_URL" ]; then
+    local payload=$(jq -n \
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)